### ISSUE TITLE
Google Maps Memory Leak When DOM is Recreated still appear.

### ISSUE THREAD

**Issue by Nikitikitavi** [allenhwkim/angularjs-google-maps#751]
Issue https://github.com/allenhwkim/angularjs-google-maps/issues/419 still exist.
I checked your example - https://rawgit.com/allenhwkim/angularjs-google-maps/master/testapp/ui-router.html#/foo and it still cause a memory leak

**Comment by allenhwkim** [comment]
It's not easy to dealing with memory with Google Maps.  If anyone has a good solution, please help.

**Comment by jlynde** [comment]
Is there any update on this issue?

**Comment by allenhwkim** [comment]
no, since this is Google maps issue, it's almost impossible to crack it.
